odc/geo/_xr_interop.py

```diff
@@ -657,6 +748,13 @@ def geobox(self) -> Optional[SomeGeoBox]:
         """Query :py:class:`~odc.geo.geobox.GeoBox` or :py:class:`~odc.geo.gcp.GCPGeoBox`."""
         return self._state.geobox
 
+    @property
```

Comment:
where did this come from? bad merge or something?

Reply:
Yeah, I think I accidently went one commit to far when running `git reset --soft HEAD~<number of commits to merge into one>`.  Although it confuses me that the same code is in the `crop` branch and `develop` but is still being detected as a diff...
https://github.com/opendatacube/odc-geo/blob/crop/odc/geo/_xr_interop.py#L755-L760
https://github.com/opendatacube/odc-geo/blob/develop/odc/geo/_xr_interop.py#L660-L665